test/test_fx_experimental.py

```diff
@@ -1623,21 +1625,40 @@ def jit_infer_type(v):
             param_names = []
             param_values = []
             fx_args = []
-            for idx, v in enumerate(arg_values):
-                if isinstance(v, torch.Tensor):
-                    param_names.append(f"arg_{idx}")
-                    param_values.append(v)
-                    fx_args.append(param_names[-1])
+
+            idx = 0
+
+            def process_arg(arg, name):
```

Comment:
why are these necessary?

Reply:
The test didn't work for arguments that were lists of tensors which is needed for the indices in the `_unsafe_masked_index` operation. I fixed the test here.